lib/elixir/lib/map.ex

```diff
@@ -166,9 +166,18 @@ defmodule Map do
 
   ## Examples
 
-      Map.values(%{a: 1, b: 2})
+      iex> Map.values(%{a: 1, b: 2})
```

Comment:
The reason we don't use `iex>` is because the order of the returned values is not guaranteed. But we could do this:

```suggestion
      iex> Map.values(%{a: 1, b: 2}) |> Enum.sort()
```